google/cloud/storage/_helpers.py

```diff
@@ -34,6 +34,12 @@
 
 _DEFAULT_STORAGE_HOST = u"https://storage.googleapis.com"
 
+# etag match parameters in snake case and equivalent header
+_ETAG_MATCH_PARAMETERS = (
+    ("if_etag_match", "If-Match"),
+    ("if_etag_not_match", "If-None-Match"),
```

Comment:
I went with "not_match" for the parameter name even though the header is "none-match" to align with the generation parameter names. Really all the HTTP stuff is an implementation detail to most users of this SDK.

Reply:
Following the generation / metageneration pattern seems right to me as well.